rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -255,11 +254,23 @@ MasterGraph::decrease_image_count()
         _remaining_count -= (_is_sequence_reader_output ? _sequence_batch_size : _user_batch_size);
 }
 
+void
+MasterGraph::calculate_cpu_num_threads(size_t shard_count)
+{
+    if (_cpu_num_threads <= 0) {
+        const unsigned DEFAULT_SMT_COUNT = 2;
```

Comment:
please use standard naming convension throughout (rename all CAP names)

Reply:
Resolved in the latest commit